Makefile

```diff
@@ -1,10 +1,12 @@
 CODE_COVERAGE_REPORT_FOLDER = $(PWD)/build/test-coverage
 CODE_COVERAGE_REPORT_NAME_UNIT = $(CODE_COVERAGE_REPORT_FOLDER)/coverage-unit-report
+VER_PKG = github.com/elastic/elastic-package/internal/version
 
 .PHONY: build
 
 build:
-	go get -ldflags "-X github.com/elastic/elastic-package/internal/version.CommitHash=`git describe --always --long --dirty` -X github.com/elastic/elastic-package/internal/version.BuildTime=`date +%s`" \
+	go install -ldflags \
+	    "-X $(VER_PKG).CommitHash=`git describe --always --long --dirty` -X $(VER_PKG).BuildTime=`date +%s` -X $(VER_PKG).Tag=`(git describe --exact-match --tags 2>/dev/null || echo '') | tr -d '\n'`" \
```

Comment:
Maybe this line can be split in multiple lines for easier readability.

Reply:
Unfortunately this is not possible due to the interaction of make string handling and `-ldflags` behaviour.

Reply:
My first guess was PKG_CONFIG as it is a Makefile.

Thanks for adjusting it.